neofetch

```diff
@@ -3202,6 +3202,49 @@ END
             # Default fallback font hardcoded in terminal-preferences.c
             [[ -z "$term_font" ]] && term_font="Monospace 12"
         ;;
+
+        "conemu-"*)
+            local ce_arg_list
+            local ce_arg_idx
+            local ce_conf
+
+            # Could have used `eval set -- "$ConEmuArgs"` instead for arg parsing but eval is evil...
+            readarray -t ce_arg_list < <(xargs -n1 printf "%s\n" <<< "${ConEmuArgs-}")
```

Comment:
There's is probably an even better way of doing this. What does the value of "$ConEmuArgs-" look like?

Reply:
When ConEmu is launched without argument, this var is empty so I tried this command for example :

```
"%ProgramFiles%\ConEmu\ConEmu64.exe" ^
    -NoUpdate ^
    -LoadCfgFile %APPDATA%\Conemu.xml ^
    -Single ^
    -Title "This is great!" ^
    -Dir "%ProgramFiles%" ^
    -run "{Bash::CygWin Bash}" ^
    -new_console
```

And it looks like :

```
$ printf '%s\n' "$ConEmuArgs"
-NoUpdate     -LoadCfgFile C:\Users\Steve\AppData\Roaming\Conemu.xml     -Single     -Title "This is great!"     -Dir "C:\Program Files"
```

Note that any argument after `-run` is ignored. According to [ConEmu's doc](https://conemu.github.io/en/ConEmuArgs.html) :

> This must be the last used switch (excepting -new_console and -cur_console).